packages/app/src/client/views/ring-health/TokenDialog.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2021 Opstrace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Box } from "client/components/Box";
+import { Dialog, DialogTitle } from "client/components/Dialog";
+import React from "react";
+
+type Props = {
+  tokens?: Array<number>,
+  onClose: () => void
+}
+
+const TokenDialog = ({ tokens, onClose }: Props) => {
+  return (
+    <Dialog open={!!tokens} onClose={onClose} aria-labelledby="simple-dialog-title">
+      <DialogTitle id="simple-dialog-title">Tokens</DialogTitle>
+      <Box pl={3} width={300} height={500}>
+        {tokens?.map(token => (
+          <div>
+            {token}
+          </div>
+        ))}
+      </Box>
+    </Dialog>
```

Comment:
The Dialog component doesn't seem to be particularly responsive, so right now it just shows a one column line of tokens. I can add manually responsiveness - unless we already have a responsive dialog.

Reply:
@MoSattler check out this example on how to set the Dialog to be responsive with a maxSize https://next.material-ui.com/components/dialogs/#optional-sizes

cc/ @terrcin (I know you've also been working with the Dialog and had responsive challenges)